examples/tutorial/x/blog/handlers.go

```diff
@@ -369,33 +369,33 @@ func (h ChangeBlogAuthorsMsgHandler) validate(ctx weave.Context, db weave.KVStor
 		return nil, nil, err
 	}
 	if obj == nil || (obj != nil && obj.Value() == nil) {
-		return nil, nil, ErrBlogNotFound()
+		return nil, nil, errors.ErrNotFound.New("blog")
 	}
 
 	blog := obj.Value().(*Blog)
 	// Check main signer is one of the blog authors
 	if findAuthor(blog.Authors, sender.Address()) == -1 {
-		return nil, nil, ErrUnauthorisedBlogAuthor(sender.Address())
+		return nil, nil, errors.ErrUnauthorized.Newf(unauthorisedBlogAuthor, sender.Address())
```

Comment:
Okay, I have now seen around 4 uses of `unauthorizedBlogAuthor`, and see why you extracted it.
Easier to change, but harder to read.

I dunna, ambivilent around it

Reply:
I reuse format strings for a reason, mainly when there are too many occurrences.
What I will do is rename it to blahBlahFormat